packages/libsodium/buildinfo.json

```diff
@@ -1,7 +1,7 @@
 {
   "single_source" : {
     "kind": "url_extract",
-    "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.9.tar.gz",
-    "sha1": "6e886fa6c7b0c0dc8a9039f49e1b04532401a6ae"
+    "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.12.tar.gz",
```

Comment:
May be we should just update the url and keep the same version. You can find the old tarball here https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.9.tar.gz